### DISCUSSION TITLE
Azure Automation Gallery Shows No Modules

### DISCUSSION THREAD

**Discussion by SonaSteve** [mithrandyr/SimplySql#118] [Q&A] [answered]
I'm trying to import SimplySql from the gallery in Azure Automation but no Contents are showing. Anyone else seen this or have any ideas? I opened a support ticket but I'm not confident that I'll get a workable response. Thoughts?

![Untitled](https://github.com/mithrandyr/SimplySql/assets/122839154/0ecc1f56-8bc7-4a8e-ad43-dc4dd000d701)

**Comment by mithrandyr** [comment]
@SonaSteve  -- Hey, it has been a while since I have had access to an azure subscription, so I don't have an easy way to try and replicate what is happening/figure out how to resolve it.

I'm assuming you have already come across this page and tried it out, but if not: [Import a module from the Modules gallery in the Azure portal](https://learn.microsoft.com/en-us/azure/automation/automation-runbook-gallery#import-a-module-from-the-modules-gallery-in-the-azure-portal).  [Here is another link](https://learn.microsoft.com/en-us/azure/automation/shared-resources/modules).

I may be able to sign up for a free Azure subscription and see if I can determine what is happening.

**Comment by mithrandyr** [comment, marked as answer]
@SonaSteve  -- ok, I figured out what you need to do -- basically on that page, click the button "Select" and it will allow you to add.. see screenshots:

**Step 1: Click on "Select"**
![image](https://github.com/mithrandyr/SimplySql/assets/16905470/44dbf576-c999-4590-9646-87ca12aa7a13)

**Step 2: Select 'Runtime Version', set to 5.1 and Click "Import"**
![image](https://github.com/mithrandyr/SimplySql/assets/16905470/2742bf95-9508-4875-9ac6-78a0254e7028)

**Comment by SonaSteve** [comment]
First, thank you for the quick responses.  So I do see what you're showing with 5.1. When I run a Runbook with 5.1, I now have it working. When I try the above to import 7.1, it shows no modules. Is there a way to use this with 7.1/7.2? Thanks again.

**Comment by mithrandyr** [comment]
The module does not support PowerShell 7 yet.  Some people have success using it (depending on which providers they need).  2.0 that will come out sometime this year, will have full support for PowerShell 7.
